arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -58,5 +58,30 @@ public async Task PutCollectionTruncateAsync(string collectionName)
             }
             await GetApiErrorException(response);
         }
+
+        public async Task<GetCollectionResponse> GetCollectionAsync(GetCollectionOptions options = null)
+        {
+            try
+            {
+                if (options != null)
+                {
+                    var response = await _transport.GetAsync(_collectionApiPath + "/" + options.CollectionName);
+
+                    if (response.IsSuccessStatusCode)
+                    {
+                        var stream = await response.Content.ReadAsStreamAsync();
+                        var collection = DeserializeJsonFromStream<GetCollectionResponse>(stream, false, true);
+                        return collection;
+                    }
+                    throw await GetApiErrorException(response);
```

Comment:
Wrap this in a `using` block and I think the rest of the method is now redundant after the comment above about changing to use a non-optional `string` argument for the `collectionName`.